.github/workflows/packaging.yml

```diff
@@ -0,0 +1,54 @@
+name: Packaging release assets
+
+on:
+  push:
+    branches:
+    - 'pre-release*'
+    - 'prerelease*'
+    - 'release-*'
+
+jobs:
+  build-deb:
```

Comment:
I think there should be multiple jobs. `build-dep` should be one, then we should have a `create-release` job and a `publish-assets` job that depend on `build-dep`. As done right now, this `build-dep` also create the release and publish the deb package. I think soon we will also have other jobs like `build-android`, `build-cydia`, etc. At that point, each of those jobs will need to be run on a different container.